pkg/operator/workloads/logs.go

```diff
@@ -106,23 +102,17 @@ func ReadLogs(appName string, workloadID string, verbose bool, socket *websocket
 			return
 		}
 
-		isEnded, err := IsWorkloadEnded(appName, workloadID)
-
-		if err != nil {
-			writeSocket(err.Error(), socket)
-			return
-		}
-		if isEnded {
-			logPrefix, err := getSavedLogPrefix(workloadID, appName, true)
+		if workloadID, ok := podLabels["workloadID"]; ok {
```

Comment:
Maybe add a comment here about why you added this if? Presumably because we won't go to cloudwatch for API logs?